pi-blaster.c

```diff
@@ -45,9 +45,11 @@ static char VERSION[] = "SNAPSHOT";
 #include <sys/mman.h>
 #include "mailbox.h"
 
-// Created new known_pins with raspberry pi list of pins
+#define MAX_CHANNELS	64
```

Comment:
Why 64? We will never get to 64 because the GPIO register we write to is a uint32.